extensionsGallery-insider.json

```diff
@@ -4958,6 +4958,71 @@
 					],
 					"statistics": [],
 					"flags": "preview"
+				},
+				{
+					"extensionId": "98",
+					"extensionName": "ads-extension-mongo-migration",
+					"displayName": "Azure Cosmos DB Migration for MongoDB",
+					"shortDescription": "This extension helps you migrate your MongoDB databases to Azure Cosmos DB for MongoDB",
```

Comment:
"assess migration of" instead of "migrate"

Reply:
done